pages/[id]/index.jsx

```diff
@@ -0,0 +1,58 @@
+import { useRouter } from "next/router";
+import BackButton from "@/components/BackButton";
+import styled from "styled-components";
+
+export default function GameDetails({ initialGames }) {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const selectedGame = initialGames.find((game) => game.id === id);
+
+  if (!selectedGame) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <>
+      <BackButton href={"/"}></BackButton>
+      <StyledArticle>
+        <StyledRowWrapper>
+          <StyledTitle>{selectedGame.title}</StyledTitle>
+          <StyledRating>Rating: {selectedGame.rating}</StyledRating>
+        </StyledRowWrapper>
+        <StyledDescription>{selectedGame.description}</StyledDescription>
+      </StyledArticle>
+    </>
+  );
+}
+
+const StyledArticle = styled.article`
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+  padding: 10px;
+  p {
+    border: solid 0.2rem black;
+    border-radius: 5px;
+    padding: 5px;
+  }
+`;
+
+const StyledRowWrapper = styled.div`
+  display: flex;
+  gap: inherit;
+  justify-content: space-between;
+`;
+
+const StyledTitle = styled.p`
+  align-content: center;
+`;
+
+const StyledRating = styled.p`
+  align-content: center;
+`;
```

Comment:
I don't think you need those styled components and the styling you currently apply with the properties won't have any effect. `align-content` can be set for a flex-container oder a grid-container and adjusts how the flex-items oder grid-items are ordered. Neither of you paragraphs is a flex- or grid-container.

If you remove the styling, everything should looks just as before.

```suggestion
```